foundation/foundation-lineage/foundation-data-lineage/foundation-data-lineage-java/pom.xml

```diff
@@ -36,6 +36,16 @@
     </build>
 
     <dependencies>
+        <dependency>
+            <groupId>jakarta.enterprise</groupId>
+            <artifactId>jakarta.enterprise.cdi-api</artifactId>
+            <version>4.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>jakarta.inject</groupId>
+            <artifactId>jakarta.inject-api</artifactId>
+            <version>2.0.1.MR</version>
```

Comment:
S: Might be worth pulling these up to the build-parent, seem to be used in three different modules.